WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsConnectedReaderView.swift

```diff
@@ -50,7 +51,7 @@ final class CardReaderSettingsConnectedReaderView: NSObject {
         )
 
         cell.batteryLevelLabel?.text = String.localizedStringWithFormat(batteryLabelFormat, batteryLevelString)
-        cell.serialNumberLabel?.text = connectedReader?.serialNumber ?? "Unknown"
+        cell.serialNumberLabel?.text = connectedReader?.serial ?? "Unknown"
```

Comment:
I presume you are already aware, because I've seen you do it in previous PRs, but just in case, all user facing strings need to be localized (in this case "Unknown". 

We usually declare those as static properties to a private enumeration (an enumeration so that it can not be instantiated)

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3784/commits/e8747e94d0a363bd4f5ddbb4e0b0d8f502979884